Makefile

```diff
@@ -55,6 +55,12 @@ else
 EXT =
 endif
 
+ifeq ($(NODE_JS),1)
```

Comment:
This environment variable `NODE_JS=1` must be set at compilation time.
It should probably be documented in `README.md`.